billboard.py

```diff
@@ -96,6 +95,27 @@ def json(self):
         return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
 
 
+class YearEndChartEntry(ChartEntry):
+    """Represents an entry (typically a single track) on a year-end chart.
+
+    Attributes:
+        title: The title of the track.
+        artist: The name of the track artist, as formatted on Billboard.com.
+            If there are multiple artists and/or featured artists, they will
+            be included in this string.
+        image: The URL of the image for the track.
+        rank: The track's position on the chart, as an int.
+        year: The chart's year, as a str.
```

Comment:
Is there a particular reason we include the year here?

Reply:
I realized that without knowing the chart it comes from, a `YearEndChartEntry` object loses its date information. I think I realized date information was missing when exporting the individual `YearEndChartEntry` objects to JSON.

With the same reasoning, we should include `date` for `ChartEntry` objects.

Does that make sense?

Reply:
I figured most users would be JSON-ifying an entire chart at a time, so including the date in each entry would be superfluous (just as including the chart name with each entry would be superfluous).

Reply:
I still think it's better to leave this out. Thoughts?

Reply:
Oh sure, I hadn't realized that was your question. I'm happy to remove it.

Reply:
Resolved with https://github.com/guoguo12/billboard-charts/pull/77/commits/95ce9b047215fc51de0cfc8de6554a1854b75781